### ISSUE TITLE
Split os release version in major and minor version number

### ISSUE THREAD

**Issue by zvonkok** [kubernetes-sigs/node-feature-discovery#210]
## Kind: Feature Request

We have a POC where different containers are run depending on the major rhel version.
Would it make sense to add this additional label to the os-release feature?

In the meantime I have added a hook that extracts the major version:

      feature.node.kubernetes.io/system-os_release.ID=rhel
      feature.node.kubernetes.io/system-os_release.MAJOR_VERSION=7
      feature.node.kubernetes.io/system-os_release.VERSION_ID=7.6
If we add a MINOR_VERSION too, users would have finer granularity on steering workkloads
to specific os-release versions, where there are strict restrictions on the versions running.

With the VERSION_ID we have to be too explicit in versioning selection (With MAJOR_VERSION coarse grained selection and additionally with MINOR_VERSION fine grained selection of nodes).

**Comment by marquiz** [comment]
If it useful, I think we could add that. It's soft of very simple addition, anyway.

I'm thinking about the actual label name, though. I would like to keep the feature "attributes" (i.e. the last part after the dot) as fields directly from the `/etc/os-release` file. Not start inventing new os-release fields, but, adding "feature sub-attributes", i.e. a new field in the label name, separated by another dot. For example:
```
system-os_release.VERSION_ID=7.6
system-os_release.VERSION_ID.major=7
system-os_release.VERSION_ID.minor=6
```

What do you think?

**Comment by zvonkok** [comment]
LGTM yep, do not invent the wheel again and again. On your side and it looks much nicer.

**Comment by marquiz** [comment]
I quickly wrote a patch, see #211

**Comment by marquiz** [comment]
Merged in #211. Closing this issue
